client/state/test/persistence.js

```diff
@@ -6,12 +6,15 @@ import { DESERIALIZE, SERIALIZE } from 'state/action-types';
 import { createReduxStore, reducer } from 'state';
 
 describe( 'persistence', () => {
-	test( 'initial state should serialize and deserialize without errors', () => {
-		const consoleSpy = jest.spyOn( global.console, 'error' ).mockImplementation( () => () => {} );
+	test( 'initial state should serialize and deserialize without errors or warnings', () => {
+		const consoleErrorSpy = jest.spyOn( global.console, 'error' ).mockImplementation( () => () => {} );
+		const consoleWarnSpy = jest.spyOn( global.console, 'warn' ).mockImplementation( () => () => {} );
+
 		const initialState = createReduxStore().getState();
 
 		reducer( reducer( initialState, { type: SERIALIZE } ), { type: DESERIALIZE } );
 
-		expect( consoleSpy.mock.calls ).toHaveLength( 0 );
+		expect( consoleErrorSpy.mock.calls ).toHaveLength( 0 );
+		expect( consoleWarnSpy.mock.calls ).toHaveLength( 0 );
```

Comment:
NIT: [`.not.toHaveBeenCalled()`](https://facebook.github.io/jest/docs/en/expect.html#tohavebeencalled) is probably a bit more expressive 🙂

Reply:
This leads to cleaner error messages as well.